### PR TITLE
Fix some broken styles in the media frame toolbar

### PR DESCRIPTION
**Pull request by goldenapples**
Updates the markup in a few of the Backbone templates and fixes some minor style issues in a few places.

The "help" link in the media frame toolbar didn't actually do anything before. Updated it so that it displays the help text tooltip, which looks to have been the original intent.

Also fixes a few minor styling inconsistencies between this view and the rest of the media frame.

### CODE REVIEW

File: assets/css/brightcove_video_connect.css [hunk 1]
```diff
@@ -78,6 +78,7 @@ div.brightcove-uploader div.brightcove-messages .updated {
 
 .brightcove-modal .brightcove .wp-filter {
   border: 0;
+  box-sizing: border-box;
```
Comment (peterwilsoncc) [inline assets/css/brightcove_video_connect.css:81]: Reviewer note: generated file

File: assets/js/src/views/toolbar.js [hunk 2]
```diff
@@ -11,13 +11,13 @@ var ToolbarView = BrightcoveView.extend(
 		events : {
 			'click .view-list' :                   'toggleList',
 			'click .view-grid' :                   'toggleGrid',
-			'click .brightcove-toolbar':           'toggleToolbar',
+			'click .brightcove-tooltip':           'toggleTooltip',
```
Comment (peterwilsoncc) [inline assets/js/src/views/toolbar.js:14]: Why are you renaming this throughout?

**Review by peterwilsoncc — COMMENTED** [review]
This looks good but wondering why you're renaming the toolbar -> tooltip throughout?

File: assets/js/src/views/toolbar.js (hunk 2 above)
Comment (goldenapples) [reply assets/js/src/views/toolbar.js:14]: The button shows or hides the tooltip. I'm not sure why it was called toolbar, but it looked like it was misnamed to me. I kept reading through the code to figure out where it was toggling a toolbar or what that would mean, and finally for my own readability renamed it as I understood it.

**Review by peterwilsoncc — APPROVED** [review]
Approving. Would be good to get another set of eyes on this from someone on FFx.

**Review by tareiking — APPROVED** [review]

**Pull request merged**